src/sql-parser/src/lexer.rs

```diff
@@ -85,6 +85,29 @@ impl Token {
             Token::Semicolon => "semicolon",
         }
     }
+
+    pub fn value(&self) -> &str {
+        match self {
+            Token::Keyword(kw) => kw.as_str(),
+            Token::Ident(val) => &*val,
+            Token::String(val) => &*val,
+            Token::HexString(val) => &*val,
+            Token::Number(val) => &*val,
+            Token::Parameter(val) => &*val,
+            Token::Op(val) => &*val,
+            Token::Star => "*",
+            Token::Eq => "=",
+            Token::LParen => "(",
+            Token::RParen => ")",
+            Token::LBracket => "[",
+            Token::RBracket => "]",
+            Token::Dot => ".",
+            Token::Comma => ",",
+            Token::Colon => ":",
+            Token::DoubleColon => "::",
+            Token::Semicolon => ";",
+        }
+    }
```

Comment:
So these two methods exist only to display errors, right? I think we should just find somewhere to `impl fmt::Display` so that we don't redundantly display both `left parenthesis` and `'('`.

Reply:
Yeah that's easy to do, I wasn't sure if you'd prefer simpler code and a possibly-simpler error message or more complex code and error messages that always have exactly the same structure.

Reply:
The `fmt::Display` on the error type is a reasonable place to impl this, IMO.

Reply:
Oh nevermind, I was working in `expected()` not `Display`, this is the only place that I can find that meaningfully could use this info, let me know if you'd like me to abstract this out to its own Displayable.